lib/message.rb

```diff
@@ -0,0 +1,35 @@
+require 'twilio-ruby'
+# require 'dotenv'
+# require 'order'
+
+account_sid = '***'
+auth_token = '***'
+client = Twilio::REST::Client.new(account_sid, auth_token)
+
+# from = '+447403930982' # Your Twilio number
+# to = '***' # Your mobile phone number
+
+# client.messages.create(
+# from: from,
+# to: to,
+# body: "Your order is being prepared! It should be with you in "
+# )
+
+class Message
+    attr_reader :twilio
+    def initialize(twilio = Twilio::REST::Client.new(ENV['TWILIO_ACCOUNT_SID'], ENV['TWILIO_AUTH_TOKEN']))
+        @twilio = twilio
+    end
+
+    def send_sms(phone_number)
+        time = Time.now
```

Comment:
You already have an instance variable for time elsewhere (in 'Order'), so I guess it depends how you're chaining classes and methods, but you might not need the repetition if you end up instantiating 'Message' from within another class. I wasn't quite sure about the way you were going to do things, but I thought you might have been planning to instantiate message inside the 'complete' method of the Order class anyway.